src/lib/hapi/plugins/rawPayloadToDataUri.js

```diff
@@ -0,0 +1,59 @@
+const getRawBody = require('raw-body')
+const encodePayload = require('@mojaloop/central-services-stream/src/kafka/protocol').encodePayload
+
+const requestRawPayloadTransform = (request, payloadBuffer) => {
+  try {
+    return Object.assign(request, {
+      payload: payloadBuffer.toString(),
+      dataUri: encodePayload(payloadBuffer, request.headers['content-type']),
```

Comment:
Are we sure this is the next place to encode?

Reply:
yip